explainability-service/src/main/java/org/kie/trustyai/service/payloads/ReconciledMetricRequest.java

```diff
@@ -0,0 +1,165 @@
+package org.kie.trustyai.service.payloads;
+
+import javax.enterprise.inject.Instance;
+
+import org.kie.trustyai.service.data.DataSource;
+import org.kie.trustyai.service.data.metadata.Metadata;
+import org.kie.trustyai.service.payloads.values.DataType;
+import org.kie.trustyai.service.payloads.values.TypedValue;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonPropertyOrder({ "protected", "favorable" })
+public class ReconciledMetricRequest {
+    private String protectedAttribute;
+    private String outcomeName;
+    private String modelId;
+
+    // this is the unique name of this specific request
+    private String requestName;
+
+    // this is the name of the metric that this request calculates, e.g., DIR or SPD
+    private String metricName;
+    private Double thresholdDelta;
+
+    private Integer batchSize;
+
+    private TypedValue privilegedAttribute;
+    private TypedValue favorableOutcome;
+    private TypedValue unprivilegedAttribute;
+
+    public ReconciledMetricRequest() {
+    };
+
+    public ReconciledMetricRequest(String protectedAttribute, String outcomeName, String modelId, String requestName, String metricName, Double thresholdDelta, Integer batchSize,
+            TypedValue privilegedAttribute, TypedValue favorableOutcome, TypedValue unprivilegedAttribute) {
+        this.protectedAttribute = protectedAttribute;
+        this.outcomeName = outcomeName;
+        this.modelId = modelId;
+        this.requestName = requestName;
+        this.metricName = metricName;
+        this.thresholdDelta = thresholdDelta;
+        this.batchSize = batchSize;
+        this.privilegedAttribute = privilegedAttribute;
+        this.favorableOutcome = favorableOutcome;
+        this.unprivilegedAttribute = unprivilegedAttribute;
+    }
```

Comment:
As far as I see it is not used, if so, can you remove it?
I'm not a fan of constructors with a lot of fields with the same type like here (`String abc, String def, String ghi`) because it is easy to swap a parameter and get a bug.
Especially if it is not used :)

Reply:
It is used in the creation of the `ReconciledDefinitionRequest` class, in the `super()` invocation